test/functional/p2p_blocksonly.py

```diff
@@ -2,15 +2,16 @@
 # Copyright (c) 2019 The Bitcoin Core developers
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
-"""Test p2p blocksonly"""
+"""Test p2p blocksonly mode & block-relay-only connections."""
```

Comment:
If understand new test well, could you document the 2 behaviors tested, namely that sender doesn't relay transaction to a block-relay-only peer and that receiver disconnect a block-relay-only peer relaying transaction ?

Reply:
hm, yeah those are the main behavioral checks for the block-relay-only connection, but there are also tests for blocks-only mode. the checks seem pretty well documented in the test logs (`self.log.info`), is there a particular reason you'd find them helpful here as well?

Reply:
IMO, it's quite nice to have a succinct high-level description of test coverage without necessary getting into the details.